music21/stream/__init__.py

```diff
@@ -6691,7 +6670,6 @@ def updateEndMatch(nInner):
             # Recurse rather than depend on the containers being Measures
             # https://github.com/cuthbertLab/music21/issues/266
             returnObj.remove(nTarget, recurse=True)
-        returnObj.coreElementsChanged()
```

Comment:
good.  -- though eventually this could get a speedup by using core commands to remove,...

...well, actually this should be rewritten eventually to use trees.